pkg/controller/consoleservice/consoleservice_controller.go

```diff
@@ -112,6 +114,10 @@ func add(mgr manager.Manager, r *ReconcileConsoleService) error {
 		log.Error(err, "Failed create singleton ConsoleService instance")
 	}
 
+	if err := monitoringv1.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "Failed adding monitoring to scheme")
```

Comment:
This should probably be done in controller-manager/main.go along with the other AddToSchemes.